Parsers/ASimDns/Tests/InfobloxBloxOne_ASimDns_ASimSchemaTester.csv

```diff
@@ -0,0 +1,110 @@
+﻿Result
+"(1) Warning: Missing recommended field [Dst]"
+"(1) Warning: Missing recommended field [EventUid]"
+"(1) Warning: Missing recommended field [SrcDomain]"
+"(1) Warning: Missing recommended field [SrcHostname]"
+"(1) Warning: Missing recommended field [TransactionIdHex]"
+"(2) Info: Missing optional alias [DomainCategory] aliasing non-existent column [UrlCategory]"
+"(2) Info: Missing optional alias [Duration] aliasing non-existent column [DnsNetworkDuration]"
+"(2) Info: Missing optional alias [Process] aliasing non-existent column [SrcProcessName]"
+"(2) Info: Missing optional alias [SessionId] aliasing non-existent column [DnsSessionId]"
+"(2) Info: Missing optional field [AdditionalFields]"
+"(2) Info: Missing optional field [DnsFlagsAuthenticated]"
+"(2) Info: Missing optional field [DnsFlagsAuthoritative]"
+"(2) Info: Missing optional field [DnsFlagsCheckingDisabled]"
+"(2) Info: Missing optional field [DnsFlagsRecursionAvailable]"
+"(2) Info: Missing optional field [DnsFlagsRecursionDesired]"
+"(2) Info: Missing optional field [DnsFlagsTruncated]"
+"(2) Info: Missing optional field [DnsFlagsZ]"
+"(2) Info: Missing optional field [DnsFlags]"
```

Comment:
this can be exracted from AdditionalExtensions

![image](https://github.com/user-attachments/assets/d21b9219-30a8-4db8-aaa4-5f756b8f6ec5)

Reply:
Done